src/main/resources/templates/template-ls.json

```diff
@@ -108,6 +108,32 @@
           "language": "RELAX-NG"
         }
       ]
+    },
+    {
+      "name": "JDT Language Server",

```

Comment:
Please move your JDT LS declaration after HTML language server to respect the sort.